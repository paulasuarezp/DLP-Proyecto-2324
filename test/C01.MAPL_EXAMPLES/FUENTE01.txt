// Perímetro de un rectángulo
int ladoA, ladoB;

read ladoA;
read ladoB;

print (ladoA + ladoB) * 2;
