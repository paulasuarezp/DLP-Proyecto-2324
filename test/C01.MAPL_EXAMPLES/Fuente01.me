//*
Ejemplo Fuente01.
Perímetro de un rectángulo
*//

class Fuente01;
	global
		vars
			ladoA, ladoB : INTEGER;
	
	create
		main; 
			
	feature	main is
		/* local
			
		do
			read ladoA;
			read ladoB;

			print (ladoA + ladoB) * 2;
	end /*main
	
end /* Fuente01

run main();