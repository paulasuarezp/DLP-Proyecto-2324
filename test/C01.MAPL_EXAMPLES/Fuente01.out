' Compilador mini Eiffel.
' Aquilino A. Juan.
' Versi√≥n 2024.2 (c)
#source "Fuente01.me"
#global ladoA: int
#global ladoB: int
#func main
#ret void
CALL main
HALT

#line 14
main:

#line 18
PUSHA 0
INI
STOREI

#line 19
PUSHA 2
INI
STOREI

#line 21
PUSHA 0
LOADI
PUSHA 2
LOADI
ADDI
PUSHI 2
MULI
OUTI

#line 21
RET 0, 0, 0
